### ISSUE TITLE
abs(cosh(x)) should simplify to cosh(x) for real x

### ISSUE THREAD

**Issue by symbolique** [sympy/sympy#11721]
Sympy 1.0 only simplifies in a limited set of cases:

``` py
>>> x = sympy.var('x', real=True)
>>> abs(sympy.cosh(x)).simplify()
Abs(cosh(x)) # bad
>>> (sympy.cosh(x) - abs(sympy.cosh(x))).simplify()
0 # good
>>> (sympy.cosh(x) + abs(sympy.cosh(x))).simplify()
cosh(x) + Abs(cosh(x)) # bad
```

**Comment by asmeurer** [comment]
I think to make it work in general `cosh(x).is_positive` should be True. It's strange that simplify works. It must be attempting some rewrites that cause it to reduce (like `rewrite(exp)`).

**Comment by rohitrango** [comment]
I would like to work on it. What is a good place to start?

**Comment by oscarbenjamin** [comment]
These examples all work fine now:
```julia
In [169]: x = Symbol('x', real=True)                                                                                                                          

In [170]: abs(cosh(x))                                                                                                                                        
Out[170]: cosh(x)

In [171]: cosh(x) - abs(cosh(x))                                                                                                                              
Out[171]: 0

In [172]: cosh(x) + abs(cosh(x))                                                                                                                              
Out[172]: 2⋅cosh(x)
```
Not sure when that was fixed...

**Comment by oscarbenjamin** [comment]
Looks like this was fixed in #13678
